src/components/login/Form.tsx

```diff
@@ -0,0 +1,47 @@
+'use client'
+import { useState } from "react";
+import Input from "./input";
+import { input } from "@/types/login/types";
+
+export default function Form() {
+    const [showPass, setShowPass] = useState(false);
+    const [loading, setLoading] = useState(false);
+
+    const emailProps: input = {
+        type: "email",
+        name: "E-mail",
+        placeholder: "Insira o seu e-mail",
+        id: "emailInput",
+    }
+
+    const passProps: input = {
+        type: "password",
+        name: "password",
+        placeholder: "Insira a sua senha",
+        id: "passwordInput",
+    }
+
+    function handleLogin(data: any) {
+        //PEGAR DADOS DO CONTEXT
+        "response = api.post({data}"
```

Comment:
É interessante manter o padrão de aspas simples, você pode instalar o plugin do Eslint no vscode para encontrar esses problemas de estilo durante o desenvolvimento.

```suggestion
        'response = api.post({data}'
```

Reply:
Ajustado!